docs/source/python/development.rst

```diff
@@ -267,6 +267,36 @@ we recommend the following:
 #. Add ``-DARROW_ORC=on`` to the CMake flags.
 #. Add ``--with-orc`` to the ``setup.py`` flags.
 
+
+Building documentation
+----------------------
+
+To build the documentation:
+
+#. install the Sphinx+doxygen packages
+
+   .. code-block:: shell
+
+          conda install --file ci/conda_env_sphinx.yml
```

Comment:
Note you have spurious leading spaces here.

Reply:
I think I've now removed them...